R/pkg/R/mllib.R

```diff
@@ -197,11 +197,10 @@ print.summary.GeneralizedLinearRegressionModel <- function(x, ...) {
   invisible(x)
   }
 
-#' Make predictions from a generalized linear model
-#'
 #' Makes predictions from a generalized linear model produced by glm() or spark.glm(),
 #' similarly to R's predict().
 #'
+#' @title predict
```

Comment:
I'm pretty sure roxgen2 only picks up the first title defined for that doc/rdname so that probably won't help us much in that case...